templates/prom-alertmanager.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>prom-alertmanager</Name>
+  <Repository>prom/alertmanager:latest</Repository>
+  <Registry>https://hub.docker.com/r/prom/alertmanager</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/prometheus/alertmanager</Project>
+  <Overview>The Alertmanager handles alerts sent by client applications such as the Prometheus
+    server. &#xD;
+    It takes care of deduplicating, grouping, and routing them to the correct receiver integrations
+    such as email, PagerDuty, OpsGenie, or many other mechanisms thanks to the webhook receiver.
+    &#xD;
+    It also takes care of silencing and inhibition of alerts.&#xD;
+    &#xD;
+    alertmanager/alertmanager.yml&#xD;
+    This configuration contains information about which channels to send to. For simplicity, we use
+    e-mail. Refer to the Alertmanager docs to learn about other channels.&#xD;
+    &#xD;
+    To configure prometheus to use these alerts, add the below to prometheus/prometheus.yml:&#xD;
+    alerting:&#xD;
+    alertmanagers:&#xD;
+    - scheme: http&#xD;
+    static_configs:&#xD;
+    - targets: &amp;lt; 'alertmanager:9093' &amp;gt;</Overview>
+  <Category>Productivity:</Category>
+  <WebUI>http://[IP]:[PORT:9093]</WebUI>
+  <TemplateURL>
+    https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/prom-alertmanager.xml</TemplateURL>
+  <Icon>
+    https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/prometheus.png</Icon>
+  <Requires />
+  <Config Name="Port" Target="9093" Default="9093" Mode="tcp" Description="" Type="Port"
+    Display="always" Required="false" Mask="false">9093
```

Comment:
There is no reason to set a value here when using Default

Reply:
Makes sense to me, going to get rid of the 9093

Reply:
Fixed